exit.c

```diff
@@ -0,0 +1,37 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <unistd.h>
+#include <sys/wait.h> 
+
+int main(int argc, char* argv[]){
+
+    for(int i=0;i<4;i++){
+        char* argument = argv[i + 1];
+        printf("History Command is %s\n", argument);
+    }
```

Comment:
This doesnt print the history?

Reply:
Should I get history from parent process? 
Need to pass 4 arg manually
/exit  1 2 3 4